src/app/tests/suites/certification/Test_TC_TCCM_3_1.yaml

```diff
@@ -21,183 +20,111 @@ PICS:
-          [1692170913.264522][3071:3073] CHIP:DMG:                                 status = 0x00 (SUCCESS),
-          [1692170913.264557][3071:3073] CHIP:DMG:                         },
-      disabled: true
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
 
     - label:
-          "Step 1: Commission DUT to TH (can be skipped if done in a preceding
-          test)."
-      verification: |
-
-      disabled: true
-
-    - label: "Step 2: TH reads from the DUT the OnMode attribute."
+          "Preconditions: The OnMode attribute is set to a non-NULL value from
+          the mode values indicated by the SupportedModes attribute."
       PICS: TCCM.S.A0003 && TCCM.S.F00
-      verification: |
-          ./chip-tool refrigeratorandtemperaturecontrolledcabinetmode read on-mode 1 1
+      command: "writeAttribute"
+      attribute: "OnMode"
+      arguments:
+          value: nonNull_OnMode_value
 
-          On TH(chip-tool), Verify that OnMode attribute value is an integer
-          Save the value as on_mode_dut and below is the sample log provided for the raspi platform, here OnMode value is 0
-
-          [1690190167.005388][16387:16389] CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0052 Attribute 0x0000_0003 DataVersion: 473867556
-          [1690190167.005447][16387:16389] CHIP:TOO:   OnMode: 0
-      disabled: true
+    - label: "TH reads the OnMode attribute from the DUT"
+      PICS: TCCM.S.A0003 && TCCM.S.F00
+      command: "readAttribute"
+      attribute: "OnMode"
+      response:
+          saveAs: on_mode_dut
 
-    - label: "Step 3: TH reads from the DUT the CurrentMode attribute."
+    - label: "TH reads the CurrentMode attribute from the DUT"
       PICS: TCCM.S.A0001 && TCCM.S.F00
-      verification: |
-          ./chip-tool refrigeratorandtemperaturecontrolledcabinetmode read current-mode 1 1
-
-          On TH(chip-tool), Verify that CurrentMode attribute is an integer.
-          Save the value as old_current_mode_dut and below is the sample log provided for the raspi platform, Here CurrentMode value is 0
+      command: "readAttribute"
+      attribute: "CurrentMode"
+      response:
+          saveAs: old_current_mode_dut
 
-          NOTE: If on_mode_dut is equal to old_current_mode_dut proceed to step 4. Else proceed to step 6.
-
-          [1690190333.843046][16430:16432] CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0052 Attribute 0x0000_0001 DataVersion: 2612443001
-          [1690190333.843064][16430:16432] CHIP:TOO:   CurrentMode: 0
-      disabled: true
-
-    - label: "Step 4: TH reads from the DUT the SupportedModes attribute."
+    - label:
+          "If the OnMode and CurrentMode attributes have the same value, proceed
+          to step 2c, Otherwise proceed to step 3a"
```

Comment:
That does not match the test plan step numbering, as far as I can tell.

Reply:
Updated.